motor/docstrings.py

```diff
@@ -662,6 +696,9 @@ async def create_x_index():
   {'_id_': {'key': [('_id', 1)]},
    'x_1': {'unique': True, 'key': [('x', 1)]}}
 
+
```

Comment:
I think this blank line should be placed below.

Reply:
Done